lib/session_setup/view/session_setup_page.dart

```diff
@@ -56,34 +56,44 @@ class SessionSetupView extends StatelessWidget {
               : null,
           body: Padding(
             padding: const EdgeInsets.all(8.0),
-            child: Column(
-              children: [
-                Text(
-                  'Choose a prompt bellow:',
-                  style: Theme.of(context).textTheme.headline3,
-                ),
-                TextField(
-                  onChanged: context.read<SessionSetupCubit>().changedQuery,
-                ),
-                Expanded(
-                  child: SingleChildScrollView(
-                    child: Wrap(
-                      children: [
-                        for (final prompt in state.promptList)
-                          PromptCard(
-                            prompt: prompt,
-                            selected: prompt == state.selectedPrompt,
-                            onTap: () {
-                              context
-                                  .read<SessionSetupCubit>()
-                                  .selectPrompt(prompt);
-                            },
-                          ),
-                      ],
+            child: Center(
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.center,
+                children: [
+                  Text(
+                    'Choose a prompt bellow:',
+                    style: Theme.of(context).textTheme.headline1,
+                  ),
+                  SizedBox(
+                    width: 400,
+                    child: TextField(
+                      onChanged: context.read<SessionSetupCubit>().changedQuery,
+                      autofocus: true,
+                      decoration: const InputDecoration(
+                        hintText: 'Filter or create a custom one here',
+                      ),
                     ),
                   ),
-                ),
-              ],
+                  Expanded(
+                    child: SingleChildScrollView(
+                      child: Wrap(
```

Comment:
Oh nice! That's what I was looking for, maybe a Wrap would be a good alternative to the Column I used in the sharing page 😄

Reply:
I changed there to use `Wrap` too :D

Reply:
Awesome thx!